render/render.js

```diff
@@ -1,16 +1,22 @@
 "use strict"
 
 var Vnode = require("../render/vnode")
+var df = require("../render/dom-for")
+var delayedRemoval = df.delayedRemoval
+var domFor = df.domFor
 
 module.exports = function($window) {
 	var $doc = $window && $window.document
-	var currentRedraw
 
 	var nameSpace = {
 		svg: "http://www.w3.org/2000/svg",
 		math: "http://www.w3.org/1998/Math/MathML"
 	}
 
+	var currentRedraw
+	var currentDOM
+	var currentRender
+
```

Comment:
Can this be moved back up so we can keep the history?

Reply:
I'd rather keep the declarations together. I like to have const first, then let, hence the placement. The history can be traced anyway, I don't understand what you want to preserve.

Reply:
Fair. Resolving this thread.